library/src/main/scala/giter8/GitRepository.scala

```diff
@@ -47,7 +47,7 @@ object GitRepository {
     val NativeUrl: Regex = "^(git[@|://].*)$".r
     val HttpsUrl: Regex  = "^(https://.*)$".r
     val HttpUrl: Regex   = "^(http://.*)$".r
-    val SshUrl: Regex    = "^(ssh://.*)$".r
+    val SshUrl: Regex    = "^ssh://(.*)$".r
```

Comment:
That has a lot of sense. I'll try to fix it tomorrow. GitRepository shouldn't be coupled by JGit implementation :+1: